porch/pkg/engine/engine.go

```diff
@@ -21,6 +21,7 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"reflect"
```

Comment:
This change seems to be from another commit in a different PR. I suspect something messed up with rebase.

Reply:
Good catch! Resolved by another rebase.